README.md

```diff
@@ -13,44 +13,52 @@ Other than tracing, Tracee is also capable of capturing files written to disk or
 
-Tracee is made of an executable that drives the eBPF code (`tracee`) and the eBPF code itself (`tracee.ebpf.o`). When the `tracee` executable is started, is will look for the eBPF code next to the executable, or in `/tmp/tracee`. If the eBPF is not found, the executable will attempt to build it (you can control this using the `--build-policy` flag).
+### Installation options
 
-You can get Tracee in any of the following ways:
-1. Download the executable from the [GitHub Releases](https://github.com/aquasecurity/tracee/releases) (`tracee.tar.gz`).
-2. Use the docker image from Docker Hub: `aquasec/tracee`.
-3. Build the executable from source using `make build`. For that you will need additional developement prerequisites such as golang and libelf.
-4. Build the executable from source in a Docker container which includes all the prerequisites, using `make build DOCKER=1`.
-5. Build the executable and the eBPF code from source, using `make all`, or in a Docker container using `make all DOCKER=1`.
+Tracee is made of an executable that drives the eBPF program (`tracee`), and the eBPF program itself (`tracee.bpf.o`). When the `tracee` executable is started, it will look for the eBPF program next to the executable, or in `/tmp/tracee`, or in a location specified in `TRACEE_BPF_FILE` environment variable. If the eBPF program is not found, the executable will attempt to build it automatically before it starts (you can control this using the `--build-policy` flag).
+
+The easiest way to get started is to let the `tracee` executable build the eBPF program for you automatically. You can obtain the executable in any of the following ways:
+1. Download from the [GitHub Releases](https://github.com/aquasecurity/tracee/releases) (`tracee.tar.gz`).
+2. Use the docker image from Docker Hub: `aquasec/tracee` (includes all the required dependencies).
+3. Build the executable from source using `make build`. For that you will need additional development tooling.
+4. Build the executable from source in a Docker container which includes all development tooling, using `make build DOCKER=1`.
+
+Alternatively, you can pre-compile the eBPF program, and provide it to the `tracee` executable. There are some benefits to this approach since you will not need clang at runtime anymore, as well as reduced risk of invoking an external program at runtime. You can build the eBPF program in the following ways:
+1. `make bpf`
+2. `make bpf DOCKER=1` to build in a Docker container which includes all development tooling.
+3. There is also a handy `make all` (and the `make all DOCKER=1` variant) which builds both the executable and the eBPF program.
+
+Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above.  
+The `aquasec/tracee:slim` image is a lighter-weight image that cannot build the eBPF program at run time, and is meant to be used when you already compiled the eBPF program beforehand.
```

Comment:
It is worth telling how to invoke it, especially how do you tell it where to find the bpf code

Reply:
you don't think "in any of the locations mentioned above" is clear?

Reply:
In the case of the slim docker image - the user can't put it in the directory of tracee executable neither under /tmp/tracee.
He should mount it, then use the TRACEE_BPF_FILE environment variable. I think it is worth explaining this

Reply:
good point